docs/tutorials/kubernetes-cluster-in-azure-with-gardener/_index.md

```diff
@@ -0,0 +1,63 @@
+---
+title: Create shoot clusters in Azure
+level: beginner
+category: Getting Started
+scope: app-developer
+---
+
+# Create shoot clusters in Azure
+
+### Prerequisites
+
+-   You need an Azure account.
+-   You have access to the Gardener dashboard and have permissions to create projects.
+
+
+1. Go to the Gardener dashboard and create a *Project*.
+
+    <img src="images/new_gardener_project.jpg">
+
+
+2. Get properties of your Azure account/Service Principal.
+    ```
+    Before you can provision and access a Kubernetes cluster on Azure, you need to add the account credentials in Gardener. 
+    Gardener needs the credentials to provision and operate the Azure infrastructure for your Kubernetes cluster.
+
+    **Ensure that the account has the `contributor` role.**
+    ```
+
+
+    -  Tenant ID
+    
+        The TenantID is also called DirectoryID - https://portal.azure.com/#blade/Microsoft_AAD_IAM/ActiveDirectoryMenuBlade/Properties.
+        <img src="images/azuregettenantid.jpg">
+
+    -  ClientID
+        Select the subscription.
+        <img src="images/azureselectsubscription.jpg">
+
+    - Select the SPN.
+        <img src="images/azureselectspn.jpg">
+
+    *Note:* The ClientID is also called ApplicationID.
```

Comment:
```suggestion
    *Note:* A service principal consist of a `ClientID` (also called `ApplicationID`) and a Client Secret
```

We could also refer here to the official Azure docs: https://docs.microsoft.com/en-us/azure/active-directory/develop/app-objects-and-service-principals